### PR TITLE
Fall 2021

### PR DESCRIPTION
**Pull request by sbidoul**

### CODE REVIEW

File: .github/workflows/test.yml.jinja [hunk 1]
```diff
@@ -48,16 +67,27 @@ jobs:
...
           persist-credentials: false
-      - run: oca_install_addons
-      - run: oca_init_test_database
-      - run: oca_run_tests
+      - name: Install addons and dependencies
+        run: oca_install_addons
+      - name: Check licenses
+        run: manifestoo -d . check-licenses
+      - name: Check development status
+        run: manifestoo -d . check-dev-status --default-dev-status=Beta
```
Comment (sbidoul) [inline .github/workflows/test.yml.jinja:83]: We may want to make this one non blocking until we fix things.

**Review by ivantodorovich — APPROVED** [review]

**Review by simahawk — APPROVED** [review]

**Comment by simahawk** [comment]
@sbidoul shall we merge and sync repo?

File: .github/workflows/test.yml.jinja (hunk 1 above)
Comment (sbidoul) [reply .github/workflows/test.yml.jinja:83]: done, using `continue-on-error: true`

**Comment by sbidoul** [comment]
FTR, there is a bunch of addons [failing the dev status check](https://github.com/shopinvader/odoo-shopinvader/runs/4077055556?check_suite_focus=true#step:6:6), including a few dependencies on Alpha modules (!)

I've not applied to all repos yet. To do so, run `pipx install copier`, `copier -f update`, git commit, `pre-commit run -a`.

**Pull request merged**